index.bs

```diff
@@ -2698,7 +2698,7 @@ dimensions of a [=client window=].
 
       browser.SetClientWindowStateParameters = {
         clientWindow: browser.ClientWindow,
-        browser.ClientWindowNamedState // browser.ClientWindowRectState
+        browser.ClientWindowNamedState / browser.ClientWindowRectState
```

Comment:
```suggestion
        (browser.ClientWindowNamedState // browser.ClientWindowRectState)
```

(seeing the failing tests) should not it be the following? since `/` for choosing between types and `//` for choosing between groups?

Reply:
Hm. I could be misreading the RFC, but as far as I can tell the only difference between `/` and `//` is the early- vs late-binding behavior; otherwise groups *are* types, right? Like, you'd only *use* `//` specifically to deal with groups, to avoid having to use `/` with parens around the groups. 

I see the parse failure, but I'm not certain that's not a bug in the parser you're using. (But I might be wrong.)

Reply:
Isn't it actually the following that we need?

```
browser.SetClientWindowStateParameters = {
  clientWindow: browser.ClientWindow,
  ( browser.ClientWindowNamedState // browser.ClientWindowRectState )
}

browser.ClientWindowNamedState = {
  state: "fullscreen" / "maximized" / "minimized"
}

browser.ClientWindowRectState = {
  state: "normal",
  ? width: js-uint,
  ? height: js-uint,
  ? x: js-int,
  ? y: js-int
}
```

Both `ClientWindowNamedState` and `ClientWindowRectState` are basically maps which would require curly braces (not rounded ones). And when listing them within `SetClientWindowStateParameters` we actually need rounded braces to correctly express the alternatives. Or do I see it wrongly?

Reply:
Yeah, just adding parens definitely works as well, regardless of how we read the spec about `/` vs `//`.

Reply:
Would you mind fixing the `ClientWindowNamedState` and `ClientWindowRectState` definitions as well as mentioned in my last comment?

Reply:
Whereby it looks like that other commands have a similar issue and that we are not consistent in the usage of `(` and `{`. So lets keep it out of this PR.